src/jit/assertionprop.cpp

```diff
@@ -68,8 +65,10 @@ void Compiler::optAddCopies()
         return;
     }
 
-    for (lclNum = 0, varDsc = lvaTable; lclNum < lvaCount; lclNum++, varDsc++)
+    for (unsigned lclNum = 0; lclNum < lvaCount; lclNum++)
     {
+        LclVarDsc* varDsc = &lvaTable[lclNum];
```

Comment:
I considered this, but not until I had already gone down the rabbit hole of `&lvaTable[lclNum]`. I don't feel particularly strongly either way, so if `LclVarDsc*` seems preferable, I'm happy to make that change.